src/main/java/org/moltimate/moltimatebackend/constant/AminoAcidType.java

```diff
@@ -0,0 +1,37 @@
+package org.moltimate.moltimatebackend.constant;
+
+import lombok.Getter;
+
+@Getter
+public enum AminoAcidType {
```

Comment:
I need something where I can pass in a 3-letter code (such as ASP) and receive its unique letter mapping